documentation/user_docs/docs/manual/Extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -191,6 +333,82 @@ The structure of the arguments to cut is still the same (see `Binning arguments`
    <manual/Generating_SQW_files:gen_sqw>`. It is possible to change these by
    setting ``ex`` and ``ez`` which are vectors lying in-plane and perpendicular
    to the plane respectively.
+   
+   
+``sphere_proj`` 2D and 1D cuts samples:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Spherical projection allows you to obtain powder average. Note that binning ranges are specified in 
+target coordinate system. Energy transfer by default is expressed in inverse Angstroms and angles 
+are in degrees.
+
+.. code-block:: matlab
+
+	data_source = fullfile(fileparts(fileparts(which(horace_init))),'demo','Fe_ei401.sqw');
+	sp_proj  = sphere_proj();
+	s2    = cut(data_source,sp_proj,[0,0.1,14],[0,180],[-180,180],[-10,4,400]);
+	plot(s2);
+
+The default constructor builds spherical projection with ``sp_proj.ez == [1,0,0]``, ``sp_proj.ex == [0,1,0]``
+and ``sp_proj.offset == [0,0,0,0]``  Cut produces:
+
+.. figure:: ../images/powder_avrg.png
+   :align: center
+   :width: 500px
+   :alt: |Q|-dE cut.
+
+   MAPS; Scattering from iron at 400meV.
+
+To the experts in the field this picture shows that the energies of phonons excitations are located under 50meV, some magnetic
+scattering is present at |Q| < 5 and spin waves at high |Q| are suppressed by magnetic form factor.
+
+Using spherical projection we can conveniently investigate the details of the particular spin wave
+presented on the images produced using linear projection above, i.e. around the scattering point :math:`[0,-1,1]`. 
```

Comment:
```suggestion
presented on the images produced using spherical projection above, i.e. around the scattering point :math:`[0,-1,1]`. 
```
unless you've abruptly changed the subject

Reply:
Using spherical projection we can conveniently investigate the details of the particular spin wave branches taken from 
the objects produced using linear projection above, i.e. around the scattering point :math:`[0,-1,1]`. 

Hope this description is more clear. I am doing spherical cut from linear cut.